style.css

```diff
@@ -3,6 +3,12 @@ body {
   padding-bottom: 2em;
```

Comment:
If you're not going to use the board, you should use some unused styles as well! 👍